Game/GameSinglePlayerScene.swift

```diff
@@ -38,10 +38,24 @@ class GameSinglePlayerScene: SKScene {
 	// MARK: - Events
 	
 	override func keyDown(with event: NSEvent) {
-//		switch event.keyCode {
-//		default:
+		//hardcoded keys for now...
+		switch event.keyCode {
+		case Keycode.leftArrow:
+			game.localPlayerInputMap[.moveLeft].activated = true
+		case Keycode.rightArrow:
+			game.localPlayerInputMap[.moveRight].activated = true
+		case Keycode.downArrow:
+			game.localPlayerInputMap[.moveDown].activated = true
+		case 0x00://US Keyboard 'a'
```

Comment:
they're not layout-specific; they're positional.  which is exactly what we want.  We shouldn't have any hard-coded keycodes, we should just use the Carbon constants.